### PR TITLE
Moving plane

### LINKED ISSUE #78: Loading bar
As a user, when I visit the welcome page, I want to see a loading bar when I click on the `Lucky Location` or `Search Location` buttons.

### PR DESCRIPTION
**Pull request by dunlapww**
- [x] Updated Files:
Updated welcome page to show plane moving back and forth on welcome screen
closes #78 
- [ ] Routes Added:

- [ ] Repo Representation: Front End

- [ ] Sad Path Tested
No testing for javascript

- [ ] Happy Path Tested
No testing for javascript

- [ ] All Tests are passing
No testing for javascript

- [ ] Updated README.md with added information
- [ ] Collabotors:
Jesse Mellinger, Will Dunlap, Todd Estes

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,41 @@
+<body>
+
+<script>
+function myMove() {
```
Comment (SageOfCode) [inline app/views/welcome/index.html.erb:4]: This is really cool! Great work.

**Review by SageOfCode — APPROVED** [review]
Excited to see this in action on Heroku.

**Review by nmking22 — APPROVED** [review]
So cool! Great job here!

File: app/views/welcome/index.html.erb (hunk 1 above)
Comment (GeorgieGirl24) [reply app/views/welcome/index.html.erb:4]: WOW!!! This is sooo cool!!!!

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -25,13 +63,24 @@
   <%= f.label :trip_duration, "Trip duration: " %>
   <%= f.number_field :trip_duration, value: 1, min: 1 %> days
 
-  <%= f.submit "Lucky Location" %>
+  <%= f.submit "Lucky Location", onclick: "myMove();"%>
```
Comment (GeorgieGirl24) [inline app/views/welcome/index.html.erb:66]: Oh, cool!!

**Review by GeorgieGirl24 — APPROVED** [review]

File: app/views/welcome/index.html.erb (hunk 1 above)
Comment (stillsheryl) [reply app/views/welcome/index.html.erb:4]: This is awesome!! Woo! ✈️

File: app/views/welcome/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+      return myMove();
+    } else {
+      pos--; //increment position by 1px
+      elem.style.left = pos + 'px';  //pixel distance from left side of parent
+    }
+  }
+}
+</script>
+
 <!-- <div class="main-container"> -->
```
Comment (stillsheryl) [inline app/views/welcome/index.html.erb:39]: I think this and the closing comment were leftover comments that can be deleted.

**Review by stillsheryl — APPROVED** [review]
Amazing work on getting this up and running!

**Pull request merged**